#goal definition
uint8 APPROACH = 0
uint8 BACKWARD = 1
uint8 DOOR_SIGNAL = 2

geometry_msgs/PoseStamped pose

uint8 goal_type
---
#result definition
bool result
---
#feedback
string feedback
